tests/bson/bson-int64-get_properties-001.phpt

```diff
@@ -4,13 +4,13 @@ MongoDB\BSON\Int64 get_properties handler (get_object_vars)
 <?php
 
 $tests = [
-    unserialize('C:18:"MongoDB\BSON\Int64":47:{a:1:{s:7:"integer";s:19:"9223372036854775807";}}'),
-    unserialize('C:18:"MongoDB\BSON\Int64":48:{a:1:{s:7:"integer";s:20:"-9223372036854775808";}}'),
-    unserialize('C:18:"MongoDB\BSON\Int64":28:{a:1:{s:7:"integer";s:1:"0";}}'),
+    '9223372036854775807',
+    '-9223372036854775808',
+    0,
 ];
 
 foreach ($tests as $test) {
-    var_dump(get_object_vars($test));
+    var_dump(get_object_vars(new MongoDB\BSON\Int64($test)));
```

Comment:
I originally did this for simplicity, but creating them outside of the loop makes potential exceptions more useful by referring directly to the item where creation failed, instead of pointing to the loop and leaving us to figure out which one failed. Updated.